packages/ui5-test-writer/templates/v4/integration/FirstJourney.js

```diff
@@ -0,0 +1,34 @@
+sap.ui.define([
+	"sap/ui/test/opaQunit"
+], function (opaTest) {
+	"use strict";
```

Comment:
Tabs are used for indentation here. Could you please change to spaces for consistency? We use spaces everywhere else.

Reply:
Done